Cargo.toml

```
@@ -128,5 +128,7 @@ time = "0.3.36"
 
 [profile.ci]
 inherits = "dev"
+debug = 0 # Don't compile debug info to reduce compilation artifact size for cache benefits.
+strip = "debuginfo" # Implied with debug = 0 since Rust 1.77, still useful for older.
```

This sentence doesn't scan to me - `older` doesn't really work as a subject. Maybe something like:
```suggestion
strip = "debuginfo" # Implied by debug = 0 since Rust 1.77 - but our MSRV is older.
```

---

Nice!

---

It's meant to be copy-pastable and in Xilem's case the MSRV is actually newer than 1.77. I adjusted the comment but made the MSRV reference a bit more conditional.

---

MSRV will be 1.82 as soon as this gets updated Peniko or Vello.

---

Maybe we should just not have the comment at all? The redundancy doesn't have any real cost